21.2 mz eud (Dashboard) (#156)
DevExpress/dotnet-eud: dashboard-for-desktop/articles/dashboard-designer/dashboard-layout.md

legacyId: 16518
---
# Dashboard Layout

This topic describes the features related to the dashboard layout in the WinForms Designer.

## Dashboard Width and Height

> The control automatically stretches or shrinks content (dashboard items) in a dashboard to fit available screen space horizontally and vertically. You can adjust the layout options and specify exact content width and height.

Reviewer: ```suggestion
The control automatically stretches or shrinks content (dashboard items) in a dashboard to fit available screen space. You can adjust the layout options and specify the width and height of content.
```

In the Ribbon menu, click the **Layout Options** button:

![|layout-options-button-in-ribbon](../../images/layout-options-button-ribbon.png)

> This invokes the **Dashboard Layout Options** dialog that allows you to specify the dashboard width and height:

Reviewer: ```suggestion
This invokes the **Dashboard Layout Options** dialog that allows you to specify the width and height of the dashboard:
```

![layout-options-dialog](../../images/layout-options-dialog.png)

* _Auto_

    The height or width of a dashboard surface fits to content.
* _Fixed_

    The height or width of a dashboard surface is set in pixels.

## Dashboard Title

> The Dashboard Title is located at the top of the dashboard surface. It can contain text, images and command buttons.

Reviewer: ```suggestion
The Dashboard Title is at the top of the dashboard surface and can contain text, images, and command buttons.
```
